settings.gradle.kts

```diff
@@ -53,6 +54,7 @@ project(":graphql-kotlin-client-generator").projectDir = file("plugins/client/gr
 project(":graphql-kotlin-sdl-generator").projectDir = file("plugins/schema/graphql-kotlin-sdl-generator")
 project(":graphql-kotlin-hooks-provider").projectDir = file("plugins/schema/graphql-kotlin-hooks-provider")
 project(":graphql-kotlin-federated-hooks-provider").projectDir = file("plugins/schema/graphql-kotlin-federated-hooks-provider")
+project(":graphql-kotlin-graalvm-metadata-generator").projectDir = file("plugins/server/graphql-kotlin-graalvm-metadata-generator")
```

Comment:
re: new `server` directory - this logic didn't make much sense to put under existing `schema` directory, directory is just used for grouping related modules 

maybe we should move schema projects under `server` functionality? or just keep those separate